docs/validators/node-configuration.md

```diff
@@ -0,0 +1,84 @@
+that won't require a code change, a new release or so on. 
+
+## Configuration files
+
+All the configuration files are by default placed into the `config` directory that resides near the node's binary. The paths can be changed 
+by using the node's CLI flags.
+
+:::important
+Note all the configuration values can be user-defined. For example, it is perfectly fine if a node operator increases the size of a cacher or sets an Elasticsearch instance, but changing the genesis total supply, for example, will lead to an inconsistent state as compared to the Network.
+:::
+
+Below you can find an example on how the configuration files should be present (at the time of writing the documentation).
+
+```
+├── api.toml                        
+├── config.toml                     
+├── economics.toml                  
+├── enableEpochs.toml               
+├── enableRounds.toml               
+├── external.toml                   
+├── gasSchedules                    
+│ ├── gasScheduleV1.toml            
+│ ├── gasScheduleV2.toml            
+│ ├── gasScheduleV3.toml            
+│ ├── gasScheduleV4.toml            
+│ ├── gasScheduleV5.toml            
+│ └── gasScheduleV6.toml            
+├── genesisContracts                
+│ ├── delegation.wasm               
+│ └── dns.wasm                      
+├── genesis.json                    
+├── genesisSmartContracts.json      
+├── nodesSetup.json                 
+├── p2p.toml                        
+├── prefs.toml                      
+├── ratings.toml                    
+├── systemSmartContractsConfig.toml 
+```
+
+- `api.toml`  holds configuration about Rest API endpoints (open or closed endpoints, logging and so on)                                                                                
+- `config.toml`  holds the main configuration of the node (storers & cachers type and size, type of hasher, type of marshaller and so on)                                                  
+- `economics.toml` holds the economics configuration (such as genesis total supply, inflation per year, developer fees and so on)                                                            
+- `enableEpochs.toml` holds a list of new features or bugfixes and the epoch when they become active                                                                                            
+- `enableRounds.toml` holds a list of new features or bugfixes and the round when they become active                                                                                            
+- `external.toml` holds external drivers' configuration (for example: Elasticsearch or event notifier)                                                                                       
+- `gasSchedules` is the directory that contains all the gas schedules to be used for SC execution, depending on activation epochs specified on external.toml->GasSchedule->GasScheduleByEpochs 
```

Comment:
`the gas consumption configuration` (without `all`)? (optional, not sure if this is better)

Reply:
fixed :pepe-pray: